src/scopes.py

```diff
@@ -4,10 +4,11 @@
 # When you use a variable in a function, it's local in scope to the function.
 x = 12
 
-def changeX():
+def changeX(x):
```

Comment:
You can change x without taking it as an argument in the function with the `global` keyword. Here's some examples https://www.programiz.com/python-programming/global-local-nonlocal-variables